src/main/java/uk/gov/hmcts/reform/sscs/model/HearingDetails.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.sscs.exception.ValidationError;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+import static uk.gov.hmcts.reform.sscs.exception.ValidationError.FACILITIES_REQUIRED_MAX_LENGTH_MSG;
+import static uk.gov.hmcts.reform.sscs.exception.ValidationError.NON_STANDARD_HEARING_DURATION_REASONS_MAX_LENGTH_MSG;
+
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@SuppressWarnings("PMD.TooManyFields")
+public class HearingDetails {
+
+    @NotNull(message = ValidationError.AUTO_LIST_FLAG_NULL_EMPTY)
+    private boolean autolistFlag;
+
+    @NotEmpty(message = ValidationError.HEARING_TYPE_NULL_EMPTY)
+    @Size(max = 40, message = ValidationError.HEARING_TYPE_MAX_LENGTH)
+    private String hearingType;
+
+    @NotNull(message = ValidationError.HEARING_WINDOW_NULL)
+    private HearingWindow hearingWindow;
+
+    @NotNull(message = ValidationError.DURATION_EMPTY)
+    @Min(value = 0, message = ValidationError.DURATION_MIN_VALUE)
```

Comment:
I don't think 0 is a rational number for this, should be 1 or more, no hearing will be 0 mins long

Reply:
this is from @dholaramgurjar, it should be update after he merge his code